crates/astria-sequencer/src/accounts.rs

```diff
@@ -0,0 +1,5 @@
+pub mod component;
```

Comment:
Note on the name of `accounts.rs`: prefer to `mv accounts.rs accounts/mod.rs`. This is because `accounts.rs` might be very far away from `accounts/` when the directory strcuture which is very confusing to navigate (like on github). Many projects prefer `mod.rs` for that reason.